src/servers/database.ts

```diff
@@ -102,6 +102,7 @@ export class DatabaseServer extends SkeletonServer {
       if (/^0x([A-Fa-f0-9]{64})$/.test(blockNumberOrHash)) {
         // Regex reference here https://ethereum.stackexchange.com/questions/34285/what-is-the-regex-to-validate-an-ethereum-transaction-hash/34286#34286
         const block_ = await this.eth_getBlockByHash(blockNumberOrHash);
+        if(block_ === null) throw Error('Block hash is temporarily unavailable');
```

Comment:
It's not that the block hash is unavailable, it's that the block information is unavailable. Let's change this to "Block is temporarily unavailable".